### PR TITLE
[WIP] Radau IIA3

### PR DESCRIPTION
**Pull request by d-netto**

### CODE REVIEW

File: src/caches/firk_caches.jl [hunk 1]
```diff
@@ -1,3 +1,103 @@
...
+  z1 = similar(u); z2 = similar(u);
+  w1 = similar(u); w2 = similar(u);
+  dw12 = similar(u, Complex{eltype(u)})
+  cont1 = similar(u); cont2 = similar(u);
+
+  fsalfirst = similar(rate_prototype)
+  k = similar(rate_prototype); k2 = similar(rate_prototype);
+  fw1 = similar(rate_prototype); fw2 = similar(rate_prototype);
+
+  J, W1 = build_J_W(alg,u,uprev,p,t,dt,f,uEltypeNoUnits,Val(true))
```
Comment (YingboMa) [inline src/caches/firk_caches.jl:80]: You only need one complex `W` matrix for 2-stage Radau IIA.

File: src/tableaus/firk_tableaus.jl [hunk 2]
```diff
@@ -1,3 +1,37 @@
...
+  T21 = T(0.9486832980505138)
+  T22 = T(0.0)
+  TI11 = T(0.0)
+  TI12 = T(1.0540925533894598)
+  TI21 = T(3.3541019662496843)
+  TI22 = T(-0.3726779962499649)
+  c1 = T2(1/3)
+  c2 = T2(1.0)
+  α  = T(1/3)
+  β  = T(-0.23570226039551578)
```
Comment (YingboMa) [inline src/tableaus/firk_tableaus.jl:28]: `α + i*β` should be the eigenvalue of `inv(A)`, so `2 + i*sqrt(2)`

**Comment by d-netto** [comment]
@YingboMa I checked the coefficients and they look fine. Still getting nonsense in the out of place. Anything that I should be looking for?

**Comment by d-netto** [comment]
oop test are passing

**Comment by YingboMa** [comment]
superseded by #1145

**Pull request closed without merging**